### PR TITLE
Server tests

### PR DESCRIPTION
**Pull request by piotr921**
Added tests for Message. 
There is only one test for Message bus, rest will be added later.
Small refactor in rest of code.

### CODE REVIEW

File: src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java [hunk 1]
```diff
@@ -6,24 +6,24 @@
...
 
-    public Message getMessageFor(int recipientId) {
+    Message getMessageFor(int recipientId) {
         Message msg = null;
-        Iterator<Message> iterator = this.messageBus.iterator();
+        Iterator<Message> iterator = messageBus.iterator();
         while (iterator.hasNext()) {
             Message current = iterator.next();
-            if(current.getRecipient() == recipientId) {
+            if (current.isToRecipient(recipientId)) {
```
Comment (arturAR) [inline src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java]: 👍

File: src/main/java/com/spanish_inquisition/battleship/common/AppLogger.java [hunk 2]
```diff
@@ -1,7 +1,11 @@
 package com.spanish_inquisition.battleship.common;
 
 import java.io.IOException;
-import java.util.logging.*;
+import java.util.logging.FileHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
```
Comment (yakisoba0) [inline src/main/java/com/spanish_inquisition/battleship/common/AppLogger.java]: Good change 👍 . Never import *

File: src/main/java/com/spanish_inquisition/battleship/server/BattleshipGame.java [hunk 3]
```diff
@@ -24,7 +22,7 @@ public void proceed() {
                 System.out.println("checking message from player " + player.getPlayerName());
                 int playerId = player.getClientId();
                 if(requestBus.haveMessageFromSender(playerId)){
-                    String message = requestBus.getMessageFromSender(playerId).getMessage();
+                    String message = requestBus.getMessageFromSender(playerId).getContent();
```
Comment (yakisoba0) [inline src/main/java/com/spanish_inquisition/battleship/server/BattleshipGame.java]: Better clarity 👍

File: src/main/java/com/spanish_inquisition/battleship/server/Message.java [hunk 4]
```diff
@@ -2,33 +2,37 @@
 
 public class Message {
     private int sender;
-    private int  recipient;
-    private String message;
+    private int recipient;
+    private String content;
```
Comment (yakisoba0) [inline src/main/java/com/spanish_inquisition/battleship/server/Message.java]: Better clarity 👍

File: src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java (hunk 1 above)
Comment (yakisoba0) [inline src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java]: From me too 👍

File: src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java [hunk 5]
```diff
@@ -66,4 +64,4 @@ public Message getMessageFromSender(int senderId) {
         messageBus.removeAll(newList);
         return msg;
     }
-}
```
Comment (yakisoba0) [inline src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java]: After we introduce packages this "package" modifiers will return to "public". But for now they can be as they are for better encapsulation.

File: src/test/java/com/spanish_inquisition/battleship/common/AdjacentFieldsCalcTest.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    @Test(dataProvider = "CoordinatesProvider")
+    public void shouldTranslateCoordinatesToIndex(int[] coordinates, int expectedIndex){
+        // When
+        int calculatedIndex = AdjacentFieldsCalc.translateCoordinatesToIndex(coordinates[0],coordinates[1]);
+
+        // Then
+        Assert.assertEquals(calculatedIndex, expectedIndex);
+    }
```
Comment (yakisoba0) [inline src/test/java/com/spanish_inquisition/battleship/common/AdjacentFieldsCalcTest.java]: Tests with DataProviders 👍

File: src/test/java/com/spanish_inquisition/battleship/server/MessageBusTest.java [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+                message.getSender(), message.getRecipient(), message.getContent()));
+        // When
+        Message getMessage = messageBus.getMessageFor(recipientId);
+
+        // Then
+        Assert.assertEquals(expectedMessage.getContent(), getMessage.getContent());
+        Assert.assertEquals(expectedMessage.getSender(), getMessage.getSender());
+        Assert.assertEquals(expectedMessage.getRecipient(), getMessage.getRecipient());
+        Assert.assertFalse(messageBus.haveMessageForRecipient(recipientId));
+    }
```
Comment (yakisoba0) [inline src/test/java/com/spanish_inquisition/battleship/server/MessageBusTest.java]: 👍

File: src/test/java/com/spanish_inquisition/battleship/server/MessageTest.java [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    @Test
+    public void shouldGetMessageContent(){
+        // When
+        String getContent = testMessage.getContent();
+
+        // Then
+        Assert.assertEquals(content, getContent);
+    }
```
Comment (yakisoba0) [inline src/test/java/com/spanish_inquisition/battleship/server/MessageTest.java]: 👍

**Pull request merged**